### ISSUE TITLE
Appointment Details Content Review from Appointments Team

### ISSUE THREAD

**Issue by ldelacosta** [department-of-veterans-affairs/va.gov-team#63219]
## About your team

- Team name: 
-	OCTO-DE product owner: Katherine Lawyer
-	Product manager: Jeff Roof and Leah De La Costa
-	Designer: Peter Russo, Ciera Maddox and Melissa Lefevre 
- Accessibility specialist:
- Content writer: 
-	Slack channel: #appointments-team
 
### Describe what you need in a few sentences:

We are requesting for the CAIA team to review the content on the appointment details page. 

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [X] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [X] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
We would like to start working on the initiative in September 2023

### Have you worked with CAIA before? 
  - [X] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** reviewing the content feedback from CAIA team 
- **Provide date if available**: xx/xx/xxxx

#### Timing for your next step: 
  - [ ] Next week
  - [X] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [X] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [X] Yes
  - [ ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** 12/31/2023
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [X] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: 
- **Collab. Cycle Ticket Opened (date)**: xx/xx/xxxx
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [X] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**:
- **User flows**:
- **Prototype or mockups**:
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:**
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [X] Danielle
- [ ] Beth
- [X] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [ ] Let’s work async and meet as needed.

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by laurwill** [comment]
Hi @ldelacosta , are you still adding supporting artifacts here with the content to review? Thanks!

**Comment by ldelacosta** [comment]
@laurwill - yes, our team is working on updating the ticket. Once we have it ready, I will post it in the slack channel. Thanks!

**Comment by Cieramaddox** [comment]
@coforma-terry @laurwill @strelichl @sara-amanda 
Hey all, 
Are there any updates on this ticket? Our team is looking to start usability tasting on the details page redesign by the end of September.

**Comment by coforma-terry** [comment]
Hi @Cieramaddox  - acknowledged - we'll be getting to this one very shortly, thanks!

**Comment by sara-amanda** [comment]
## CAIA Update: Research Ticket Creation
@Cieramaddox for the research portion can you fill out the separate research ticket? We can use that ticket to plan on who from CAIA's Accessibility Team will attend your sessions, etc. 🥳 **_Thank you for reaching out to us!_** We are happy to assist with your AT sessions, and provide AT-related user research feedback.  
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [x]  I went ahead and marked this off in the body of the ticket, based on your comment above. 😄 

cc: @SarahKay8 @coforma-terry

**Comment by Cieramaddox** [comment]
@sara-amanda 
Ah okay, I may have filed this ticket wonky. So, we wont need accessibility support for our user research, but we would love it if y'all could just look over our designs and bring any accessibility concerns to light.

**Comment by laurwill** [comment]
Hi @Cieramaddox , I've added comments with content Qs and suggestions to the Figma screens. 

My biggest open question is around claim exams — those have lots of specific info that we outline [on this page](https://www.va.gov/disability/va-claim-exam/), that the claim exam detail page doesn't reflect. I'm not sure of the best approach here. Have you considered linking out to the static claim exam page for more information, or are you trying not to send people outside the appointments tool from these screens?

**Comment by Cieramaddox** [comment]
@laurwill 
Okay thanks for bringing this up. Claim exams are the newest appt type that we have and will certainly be ironing out some kinks.

**Comment by outerpress** [comment]
@ldelacosta @Cieramaddox is this one done? I think we can close

**Comment by laurwill** [comment]
@outerpress if you all are going to update this work based on testing findings, we can leave this intake open and hang new tickets related to it as needed — but if you're totally done iterating on the details pages for now, we can close. Let us know which makes sense here!
